test/tests/functional/pbs_mom_hook_sync.py

```diff
@@ -275,7 +275,7 @@ def test_momhook_to_serverhook_with_restart(self):
         # outcome, as send action occurs after the delete action.
         self.server.log_match(
             'Node;%s.*;' % (self.momB.hostname,) +
-            'Received IS_HELLOSVR from Mom',
+            'Hello from MoM',
```

Comment:
not sure, but i think you need to add `.*` after `MoM`, no?

Reply:
I confirmed that its working!